core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -136,7 +136,8 @@ private[spark] class CoarseMesosSchedulerBackend(
       sc.sparkUser,
       sc.appName,
       sc.conf,
-      sc.ui.map(_.appUIAddress))
+      sc.conf.getOption("spark.mesos.scheduler.webui.url").orElse(sc.ui.map(_.appUIAddress))
```

Comment:
Can you call this "driver" instead of "scheduler"?  The dispatcher is also a Mesos "scheduler", so this is ambiguous.